.gitignore

```diff
@@ -4,6 +4,8 @@
 .env
 .DS_Store
 venv/
+.venv
```

Comment:
There is an extra space after `.venv` which might cause it to not be ignored properly. Consider removing the space to ensure `.venv` is correctly ignored.